spring-batch-docs/asciidoc/monitoring-and-metrics.adoc

```diff
@@ -0,0 +1,69 @@
+:batch-asciidoc: ./
+:toc: left
+:toclevels: 4
+
+[[monitoring-and-metrics]]
+
+== Monitoring and metrics
+
+Since version 4.2, Spring Batch provides support for batch monitoring and metrics
+based on link:$$https://micrometer.io/$$[Micrometer]. This section describes
+which metrics are provided out-of-the-box and how to contribute custom metrics.
+
+[[built-in-metrics]]
+
+=== Built-in metrics
+
+Metrics collection does not require any specific configuration. All metrics provided
+by the framework are registered in
+link:$$https://micrometer.io/docs/concepts#_global_registry$$[Micrometer's global registry]
+under the `spring.batch.` prefix. The following table explains all metrics in details:
+
+|===============
+|__Metric Name__|__Type__|__Description__
+|spring.batch.job|TIMER|Duration of job execution
```

Comment:
I would use the code font for the Metric Name and Type values (not the headers but the contents of each row.